### ISSUE TITLE
perf(qwen35): profile and accelerate Qwen3.6 first-token prefill/decode

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#438]
## Problem

The real Qwen3.6-27B pure-fak smoke now reaches generation, but first-token latency is far from the llama.cpp bar and lacks a phase breakdown.

Measured on Apple M3 Pro / 36 GB, 2026-06-19:

```text
prompt_tokens=22
prefill: 22 tok in 127.48s (0.2 tok/s)
cached qwen3_5 decode: 1 tok in 3.92s (0.3 tok/s)
```

The comparable llama.cpp Metal reference in `fak/QWEN36-PARITY-RESULTS.md` is:

```text
prefill: 51.55 tok/s
decode: 7.29 tok/s
```

Today we cannot tell from the command output how much time is in Gated-DeltaNet projections, depthwise conv/scan, gated RMSNorm, full-attention layers, MLP, activation quantization, or output head.

## Acceptance

- [ ] Add `FAK_QPROFILE` or an equivalent profiler mode for Qwen35 cached prefill/decode that reports per-layer and per-op timings.
- [ ] Include at minimum: embedding, input/post norms, GDN qkv/z/a/b projections, conv1d, recurrent delta scan, GDN out projection, full-attention q/k/v/gate/o path, MLP gate/up/down, final norm, logits/head.
- [ ] Run the 22-token Qwen3.6-27B smoke and publish the profile table in `FAK-NATIVE-QWEN35-RESULTS.md` or a linked experiment artifact.
- [ ] Use the profile to split follow-up implementation tickets for the top two bottlenecks, with before/after tok/s gates.

## Notes

This is a profiling/optimization child of #88 and #92. It should not claim correctness parity; #93 owns the real-artifact oracle.

---
_Migrated from an internal tracker._

**Comment by anthony-chaudhary** [comment]
Resolved by `1fbd792` (resolving commit). Closed by the DOS dispatch loop's close-resolved arm, witnessed via `dos commit-audit` (verdict OK / diff-witnessed). Reopen if this does not fully resolve it.
